CsCore/PlainNetClassLib/src/Plugins/CsCore/com/csutil/algorithms/images/Mat.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+using StbImageSharp;
+
+namespace com.csutil.algorithms.images
+{
+    public class Mat<T>
```

Comment:
Note to myself:
This class here could also be relevant for @XANGEnino s PR, see https://github.com/cs-util-com/cscore/pull/125/files#r1483888046 to make use of it there

Both PRs are also adding the same StbImageSharp library